plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_alerts/task.py

```diff
@@ -61,6 +61,15 @@ def run(self, params={}, state={}, custom_config: dict = {}):  # pylint: disable
 
             return response, state, has_more_pages, 200, None
 
+        except APIException as error:
+            return (
+                [],
+                {},
```

Comment:
Do we want to return an empty dict for state here? or should just return `existing_state` ?

Reply:
existing state isn't used for anything in an error case in pif 🤷

Reply:
yeah fair enough, just a np more than anything as that's the (albeit redundant) pattern that we usually follow